components/crud-web-apps/jupyter/backend/apps/common/status.py

```diff
@@ -8,54 +8,75 @@
 
 def process_status(notebook):
     """
-    Return status and reason. Status may be [running|waiting|warning|error]
+    Return status and reason. Status may be [running|waiting|warning]
     """
-    # Check if the Notebook is stopped
     readyReplicas = notebook.get("status", {}).get("readyReplicas", 0)
-    metadata = notebook.get("metadata", {})
-    annotations = metadata.get("annotations", {})
-
-    if STOP_ANNOTATION in annotations:
-        if readyReplicas == 0:
-            return status.create_status(
-                status.STATUS_PHASE.STOPPED,
-                "No Pods are currently running for this Notebook Server.",
-            )
-        else:
-            return status.create_status(
-                status.STATUS_PHASE.TERMINATING, "Notebook Server is stopping."
-            )
+    creationTimestamp = notebook["metadata"]["creationTimestamp"]
```

Comment:
Let's rename this var to `creation_timestamp`. Python is using snake_case for var names 
https://peps.python.org/pep-0008/#descriptive-naming-styles